dependencies/pom.xml

```diff
@@ -28,7 +28,6 @@
     <module>javamail</module>
     <module>jaxb</module>
     <module>jcifs</module>
-    <module>jersey</module>
     <module>jersey-client</module>
```

Comment:
With these changes, the only things left using jersey-client-dependencies appear to be features/ticketing and features/resource-cli. Does that sound correct?